<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarketListView</class>
 <widget class="QDialog" name="MarketListView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1234</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>10</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="filterMarketInfoButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>FilterMarketInfo</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>201</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="GetRegions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>GetRegions</string>
   </property>
  </widget>
  <widget class="QListWidget" name="productListWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>371</width>
     <height>601</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="productTextBrowser">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>90</y>
     <width>601</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="productHistoryTextBrowser">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>490</y>
     <width>601</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Loaded</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemStatusLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Loaded</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>60</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadMarketInfoButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>56</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>LoadMarketInfo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MarketListView</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MarketListView</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
